Find a functionally equivalent code:android.print.PrinterCapabilitiesInfo.getMediaSizes:COMMENT
Method Modifier: public      
Comment:/**
 * Gets the supported media sizes.
 *
 * @return The media sizes.
 */

@@ -1,3 +1,3 @@
 {
-    return mMediaSizes;
+    return Collections.unmodifiableList(mMediaSizes);
 }

